[TECH] Mise à jour des dépendences certifs

aceol opened the pull request:
## :unicorn: Problème
Les dependences sont pas à jour

## :robot: Proposition
On met à jour...

## :rainbow: Remarques
- Les MAJ de ember-data sont chaotique (bcp de warnings), je ne suis pas allé jusque 4.12
- Revert du "before" https://github.com/1024pix/pix/pull/5236 ca n'a pas d'impact
- La montée de version de ember-data ajoute des warning/depreciations, en particulier https://deprecations.emberjs.com/v4.x/#toc_ember-polyfills-deprecate-assign
Ce warning apparait de nombreuses fois mais semble etre uniquement lié à `ember-inputmask` utilisé unquement pour la date de naissance d'candidat dans la modale d'ajout. On pourrait soit remplacer cette lib par ember-inputmask-5 (mais qui ne fait pas exactement ça) ou ne pas avoir de mask (soit champs libre, soit datepicker)

## :100: Pour tester
Se ballader sur l'app

aceol committed:
:recycle: api: seeds: Make started session finalizable

aceol committed:
:arrow_up: certif: Update ember-simple-auth

the initializer for ember-simple-auth is not recognized, so we can't use the "before". Its removal doesnt seem to have an impact

pix-bot-github commented:
Une fois les applications déployées, elles seront accessibles via les liens suivants :
  * [App (.fr)](https://app-pr6990.review.pix.fr)
  * [App (.org)](https://app-pr6990.review.pix.org)
  * [Orga (.fr)](https://orga-pr6990.review.pix.fr)
  * [Orga (.org)](https://orga-pr6990.review.pix.org)
  * [Certif (.fr)](https://certif-pr6990.review.pix.fr)
  * [Certif (.org)](https://certif-pr6990.review.pix.org)
  * [1D](https://1d-pr6990.review.pix.fr)
  * [Admin](https://admin-pr6990.review.pix.fr)
  * [API](https://api-pr6990.review.pix.fr/api/)
  * [Audit Logger](https://pix-audit-logger-review-pr6990.osc-fr1.scalingo.io/api/)

Les variables d'environnement seront accessibles via les liens suivants :
  * [scalingo front](https://dashboard.scalingo.com/apps/osc-fr1/pix-front-review-pr6990/environment)
  * [scalingo api](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-review-pr6990/environment)
  * [scalingo audit-logger](https://dashboard.scalingo.com/apps/osc-fr1/pix-audit-logger-review-pr6990/environment)

aceol committed:
:arrow_up: certif: update ember-data

aceol committed:
:rotating_light: certif: Remove faker warning (should remove faker)

aceol committed:
:arrow_up: certif: update ember-data 4.6

aceol committed:
:arrow_up: certi: update stylelint + ember-cli-clipboard

aceol committed:
:arrow_up: certif: Update tracked-build-ins + sass

aceol committed:
:arrow_up: certif: babel

aceol committed:
:arrow_up: certif: Update formatjs/intl + @fortawesome/ember-fontawesome

aceol committed:
:arrow_up: certif: update p-queue

aceol committed:
:arrow_up: certif: update prettier

xav-car reviewed (approved):
malgré le petit conflit à gérer . ça me semble ok ( après je connais pas tout les contextes certif )

mariannebost commented:
Tests func :
- création d'une session ✅ 
- ajout candidats par ods, par modale ✅ 
- espace surveillant ✅ 
- finalisation avec incident : ✅ 
- version anglaise : ✅ 
- import en masse : ✅ 
- ajout d'un referent : ✅ 
- inscription sco : ✅

aceol committed:
:arrow_up: certif: Update ember-data 4.10

aceol committed:
:bug: certif: Manage 400 instead of 422

The backend return 400 instead of 422. Also new version of ember data does not keep the 422 status
